crates/autopilot/src/lib.rs

```diff
@@ -569,22 +569,22 @@ pub async fn main(args: arguments::Arguments) -> ! {
         args.limit_order_price_factor
             .try_into()
             .expect("limit order price factor can't be converted to BigDecimal"),
-        true,
+        !args.enable_colocation,
```

Comment:
Why is auction not stored in case of colocation?

Reply:
Because the colocation run loop (run_loop.rs) stores the auction instead.

Reply:
Thanks.